mdaviz/app.py

```diff
@@ -74,16 +72,12 @@ def main():  # for future command-line options
 
     # Ensure the path is absolute (starts with a "/")
     if not directory.startswith("/"):
```

Comment:
This `.startswith("/")` makes the program depend on specific filesystems.  Won't work on Windows.

Reply:
Maybe compare `str(directory.absolute())` with `str(directory)`?

Reply:
created #141